README.md

```diff
@@ -1 +1,10 @@
-# csdt-kravchukoi-2122
\ No newline at end of file
+# csdt-kravchukoi-2122
```

Comment:
delete main branch, create develop and set it as default

Reply:
Default branch is 'develop' now